DEVDOCS-3882: Update b2b-edition.md (#1485)
bigcommerce/dev-docs: docs/partner-apps/bundleb2b/b2b-edition.md

# B2B Edition 

BigCommerce offers optional business-to-business (B2B) functionality with BundleB2B (B3), allowing businesses to streamline online operations and improve the experience for BigCommerce store owners and their B2B customers.

The B2B Edition is a packaged offering of the BigCommerce Enterprise plan, B3, and [six BigCommerce themes optimized for B3](https://support.bigcommerce.com/s/article/B2B-Edition?language=en_US#themes).

B3 offers multiple B2B feature enhancements to your BigCommerce store, including the following:

- Customer account hierarchy with user roles and permissions

> - Shared shopping lists and buy again functionality

Reviewer: ```suggestion
- Shared shopping lists and "buy again" functionality
```

- Sales representative enablement tools
- Quote management
- Invoice & payment management

## Installing B3 

> The B3 admin panel is accessible from the My Apps section of your store's dashboard. The client-facing portions of the app are loaded dynamically with JavaScript.

Reviewer: ```suggestion
The B3 admin panel is accessible from the **My Apps** section of your store's control panel. The client-facing portions of the app are loaded dynamically with JavaScript.
```

# B2B Edition 

> The BigCommerce platform offers BundleB2B (B3) functionality which allows businesses to streamline online operations and improve experience for BigCommerce store owners and their customers.

Reviewer 2: ```suggestion
BigCommerce offers optional business-to-business (B2B) functionality with BundleB2B (B3), allowing businesses to streamline online operations and improve the experience for BigCommerce store owners and their B2B customers.
```

- We use the "B2B" abbreviation below without defining it, so I added that part back in.

> The BigCommerce platform offers BundleB2B (B3) functionality which allows businesses to streamline online operations and improve experience for BigCommerce store owners and their customers.

Reviewer 2: ```suggestion
BigCommerce offers optional business-to-business (B2B) functionality with BundleB2B (B3), allowing businesses to streamline online operations and improve the experience for BigCommerce store owners and their B2B customers.
```

- We use the "B2B" abbreviation below without defining it, so I added that part back in.

The B2B Edition is a packaged offering of the BigCommerce Enterprise plan, B3, and [six BigCommerce themes optimized for B3](https://support.bigcommerce.com/s/article/B2B-Edition?language=en_US#themes).

B3 offers multiple B2B feature enhancements to your BigCommerce store, including the following:

- Customer account hierarchy with user roles and permissions
- Shared shopping lists and buy again functionality
- Sales representative enablement tools
- Quote management
- Invoice & payment management

> ## Installing B3 

Reviewer 2: I don't think anything about this section changed as a result of BC acquiring BundleB2B. This section doesn't really describe how to install B3, and the parts about the dashboard being an iFrame while the client portion is dynamically loaded are important to the rest of the doc.

> B3 offers multiple enhancements to your BigCommerce store, including:

Reviewer: ```suggestion
B3 offers multiple enhancements to your BigCommerce store, including the following:
```

- Customer account management
- All-in-one ordering tools

> - Quoting management

Reviewer: ```suggestion
- Quote management
```

With B3, you can customize the placement of B3 app elements, display text, styling, and checkout configurations. You also have access to the lifecycle methods for many B3 modules to inject your own JavaScript functions.

Additionally, you can use the B3 REST API to create, read, update, and delete items such as orders, companies, addresses, payments, sales reps, and users. This API allows you to build your own services or integrate B3 with third-party business tools, such as Salesforce.

> This section gives details on how to edit the DOM, CSS, and display text, and how to define JavaScript lifecycle methods. It assumes you are familiar with editing Stencil themes locally and pushing them to a sandbox or production store using the Stencil CLI.

Reviewer: @traci707 the reason I went with `how to edit the DOM, CSS, and display text, and how to define JavaScript lifecycle methods` rather than `how to edit the DOM and CSS, display text, and define JavaScript lifecycle methods` is that the verbs `edit` and `define` should be in the same form. This is a list of two and an additional thing, disguised as a list of three.

I agree this is clunky. Other suggestions? I'll puzzle it over later & would love your input in the mix.

> This section gives details on how to edit the DOM, CSS, and display text, and how to define JavaScript lifecycle methods. It assumes you are familiar with editing Stencil themes locally and pushing them to a sandbox or production store using the Stencil CLI.

Author: @slsriehl We can leave it as it is. It's not a big deal.
